tests/unit/cassettes/test_sca_scan_diff.yaml

```diff
@@ -0,0 +1,194 @@
+interactions:
```

Comment:
This file reaise an error from the linter in the CI. You should run a pre-commit on it.